src/addrman.h

```diff
@@ -176,6 +177,20 @@ friend class CAddrManTest;
     mutable RecursiveMutex cs;
 
 private:
+    //! Serialization versions.
+    enum Format : uint8_t {
+        V0_HISTORICAL = 0,    //!< historic format, before commit e6b343d88
+        V1_DETERMINISTIC = 1, //!< for pre-asmap files
+        V2_ASMAP = 2,         //!< for files including asmap version
+        V3_BIP155 = 3,        //!< same as V2_ASMAP plus addresses are in BIP155 format
+    };
+
+    //! The initial value of a field that is incremented every time an incompatible format
+    //! change is made. This is 32 instead of 0 because we overtook the "key size" field
+    //! which was 32 historically and we don't want to start with 0 and bump into 32 at
+    //! some point in the future.
+    static constexpr uint8_t m_incompatibility_base = 32;
```

Comment:
We usually use ALL_CAPS for constant names.

Reply:
Changed to `INCOMPATIBILITY_BASE`.